services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -158,7 +194,41 @@ public RouteTableWebJson getOrCreateVpcRouteTable(@PathVariable String projectid
             RestPreconditionsUtil.verifyParameterNotNullorEmpty(projectid);
             RestPreconditionsUtil.verifyResourceFound(projectid);
 
-            routetable = this.routerService.getOrCreateVpcRouteTable(projectid, vpcid);
+            routetable = this.routerService.getVpcRouteTable(projectid, vpcid);
+
+        } catch (ParameterNullOrEmptyException e) {
+            throw e;
+        } catch (CanNotFindVpc e) {
+            logger.log(Level.WARNING, e.getMessage() + " : " + vpcid);
+            throw e;
+        } catch (DatabasePersistenceException e) {
+            throw e;
+        }
+
+        return new RouteTableWebJson(routetable);
+    }
+
+    /**
+     * Create VPC default route table
+     * @param projectid
+     * @param vpcid
+     * @return
+     * @throws Exception
+     */
+    @RequestMapping(
+            method = POST,
+            value = {"/project/{projectid}/vpcs/{vpcid}/vpcroutetable"})
+    @DurationStatistics
+    public RouteTableWebJson createVpcRouteTable(@PathVariable String projectid, @PathVariable String vpcid) throws Exception {
+
+        RouteTable routetable = null;
+
+        try {
+            RestPreconditionsUtil.verifyParameterNotNullorEmpty(vpcid);
+            RestPreconditionsUtil.verifyParameterNotNullorEmpty(projectid);
+            RestPreconditionsUtil.verifyResourceFound(projectid);
+
+            routetable = this.routerService.createVpcRouteTable(projectid, vpcid);
```

Comment:
@DavidLiu506 If routetable is already exist, return `409 conflict`.

Reply:
Add to createVpcRouter function in RouterServiceImpl

Reply:
ok